dev/update-blueprint-dependencies.js

```diff
@@ -189,4 +193,8 @@ if (module === require.main) {
   }
 }
 
+function isEmberDataPackage(packageName) {
+  return packageName.includes('ember-data');
```

Comment:
We could add the `@warp-drive/core-types` package here as well, but I don't think it's needed. We only used the beta because there was no stable release out yet.